### ISSUE TITLE
Training Log?

### ISSUE THREAD

**Issue by XinWangg** [zhongyy/Face-Transformer#5]
Hi!
Can you share your training log?

**Comment by XinWangg** [comment]
@khawar512 Yes, It takes a lot of time to train, and when I train the model, I found that the loss may not drop as quickly as CNN. So I want to ask for the log to see whether there is something wrong.

**Comment by zhongyy** [comment]
Please refer to the training logs for ViT-P10S8.

**Comment by XinWangg** [comment]
> Please refer to the training logs for ViT-P10S8. I have uploaded a 'log' folder.

Thanks!

**Comment by khawar512** [comment]
@XinWangg  you are right. It takes a lot of time Thank you @zhongyy

**Comment by Admer996** [comment]
> @XinWangg The author shared a trained model not a log and he also shared all steps but literally training on 1 million images takes a lot of time and GPUs as well. I am trying to figuring out to train on small dataset.

@khawar-islam Hello, is it possible to train on a small dataset and what is the result?
